### PR TITLE
Add bare metal OS requirements

### PR DESCRIPTION
**Pull request by chrisdoherty4**
Also update the NPM arg so we can run `make server`.

### CODE REVIEW

File: docs/content/en/docs/getting-started/baremetal/bare-prereq.md [hunk 1]
```diff
@@ -30,43 +30,52 @@ The compute hardware you need for your Bare Metal cluster must meet the followin
...
 
 When upgrading without an extra machine, keep in mind that your control plane and your workload must be able to tolerate node unavailability. When upgrading with extra machine(s), you will need additional temporary machine(s) for each control plane and worker node grouping. Refer to [Upgrade Bare Metal Cluster]({{< relref "../../clustermgmt/cluster-upgrades/baremetal-upgrades/" >}}) and [Advanced configuration for rolling upgrade]({{< relref "../../clustermgmt/cluster-upgrades/baremetal-upgrades/#advanced-configuration-for-rolling-upgrade" >}}).
 
->**_NOTE_**: For single node clusters that require an additional temporary machine for upgrading, if you don't want to set up the extra hardware, you may recreate the cluster for upgrading and handle data recovery manually.
+> **_NOTE_**: For single node clusters that require an additional temporary machine for upgrading, if you don't want to set up the extra hardware, you may recreate the cluster for upgrading and handle data recovery manually.
```
Comment (chrisnegus) [inline docs/content/en/docs/getting-started/baremetal/bare-prereq.md:42]:
```suggestion
> **_NOTE_**: For single-node clusters that require an additional temporary machine for upgrading, if you don't want to set up the extra hardware, you may recreate the cluster for upgrading and handle data recovery manually.
```
nit

**Review by chrisnegus — COMMENTED** [review]
One nit.

**Review by vivek-koppuru — APPROVED** [review]

File: docs/content/en/docs/getting-started/baremetal/bare-prereq.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Set up an Administrative machine as described in Install EKS Anywhere.
 
 ## Compute server requirements
 
-The minimum number of physical machines needed to run EKS Anywhere on bare metal is 1. To configure EKS Anywhere to run on a single server, set controlPlaneConfiguration.count to 1, and omit workerNodeGroupConfigurations from your cluster configuration. 
+The minimum number of physical machines needed to run EKS Anywhere on bare metal is 1. To configure EKS Anywhere to run on a single server, set `controlPlaneConfiguration.count`to 1, and omit `workerNodeGroupConfigurations` from your cluster configuration.
```
Comment (chrisnegus) [inline docs/content/en/docs/getting-started/baremetal/bare-prereq.md:20]:
```suggestion
The minimum number of physical machines needed to run EKS Anywhere on bare metal is 1. To configure EKS Anywhere to run on a single server, set `controlPlaneConfiguration.count` to 1, and omit `workerNodeGroupConfigurations` from your cluster configuration.
```

File: docs/content/en/docs/getting-started/baremetal/bare-prereq.md [hunk 3]
```diff
@@ -30,43 +30,52 @@ The compute hardware you need for your Bare Metal cluster must meet the followin
 * Memory: 8GB RAM
 * Storage: 25GB
 
+## Operating system requirements
+
+If you intend on using a non-Bottlerocket OS you must build it using image builder. See the [OS Management Artifacts]({{< relref "../../osmgmt/artifacts#bare-metal-artifacts" >}}) page for help building the OS.
```
Comment (chrisnegus) [inline docs/content/en/docs/getting-started/baremetal/bare-prereq.md:35]:
```suggestion
If you intend on using a non-Bottlerocket OS you must build it using `image-builder`. See the [OS Management Artifacts]({{< relref "../../osmgmt/artifacts#bare-metal-artifacts" >}}) page for help building the OS.
```

**Review by chrisnegus — COMMENTED** [review]
Just a couple of small suggestions.

**Pull request merged**